packages/nuxt/src/core/plugins/dev-only.ts

```diff
@@ -9,7 +9,8 @@ interface DevOnlyPluginOptions {
 }
 
 export const DevOnlyPlugin = createUnplugin((options: DevOnlyPluginOptions) => {
-  const DEVONLY_COMP_RE = /<(?:dev-only|DevOnly)>[^<]*(?:<template\s+#fallback>(?<fallback>[\s\S]*?)<\/template>)?[\s\S]*?<\/(?:dev-only|DevOnly)>/g
+  const DEVONLY_COMP_RE = /<(?:dev-only|DevOnly)>[\s\S]*?<\/(?:dev-only|DevOnly)>/g
+  const FALLBACK_SLOT_RE = /<template\s+#fallback>(?<fallback>[\s\S]*?)<\/template>/
```

Comment:
This regex is working but only if there's no multiple `<template #fallback>` within the string. 
For example: 
````html
 <DevOnly>
    <template #fallback>
      <div><!-- empty div for flex.justify-between --></div>
      <ClientOnly>
        <template #fallback>
          <div>test</div>
        </template>
      </ClientOnly>
    </template>
</DevOnly>
````
will get you
````html
<div><!-- empty div for flex.justify-between --></div>
      <ClientOnly>
        <template #fallback>
          <div>test</div>
````
Also if you set a fallback somewhere before the `<template #fallback>` of devonly, it will catch what's inside the `<template #fallback>.

````html
 <DevOnly>
    <ClientOnly>
      <template #fallback>
        <div>test</div>
      </template>
    </ClientOnly>
    <template #fallback>
      <div><!-- empty div for flex.justify-between --></div>
    </template>
</DevOnly>
````
will get you
````html
      <template #fallback>
        <div>test</div>
      </template>
````

but i think the risk of someone using another `<template #fallback>` should be very low 😅. Yet, there's still a risk of someone doing it (is this murphy's law ?).

Reply:
i didn't forgot to submit the review this time :joy:

Reply:
Ooh that's a very good point. Let me brainstorm this

The thing with `ultrahtml` is that I'd think of using the sanitizer utils it provides (may increase dependency bundle) but that would also involve regex at a point while this is fast and simple so currently progressing on this

Reply:
Gone for a hybrid approach that allowed this to be done without a walker and needing to be async 😄